api/all.yaml

```diff
@@ -2792,6 +2899,108 @@ components:
+          description: The ID of the table to commit to. In the context of UC, this is the UUID that uniquely identifies a table.
+        table_uri:
+          type: string
+          description: |
+            The URI of the storage location of the table. If the table_id exists but the table_uri is 
+            different from the one previously registered (e.g., if the client moved the table), the request will fail.
+        commit_info:
+          $ref: '#/components/schemas/CommitInfo'
+        latest_backfilled_version:
+          type: integer
+          format: int64
+          description: |
+            The highest version of the commits that have been backfilled for this table; meaning UC no longer 
+            needs to keep track of commits of versions <= this version.
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+      required:
+        - table_id
+        - table_uri
+    Metadata:
+      description: Represents a Delta metadata action (see https://github.com/delta-io/delta/blob/master/PROTOCOL.md#change-metadata for more information).
+      properties:
+        description:
+          type: string
+          description: The table comment.
+        schema:
+          $ref: '#/components/schemas/ColumnInfos'
+        properties:
+          $ref: '#/components/schemas/CommitMetadataProperties'
+    CommitMetadataProperties:
+      description: The properties of the table. Updated by UC if set.
+      type: object
+      properties:
+        properties:
+          $ref: '#/components/schemas/SecurablePropertiesMap'
+    ColumnInfos:
+      type: object
+      description: The schema of the table.
+      properties:
+        columns:
+          type: array
+          items:
+            $ref: '#/components/schemas/ColumnInfo'
+          maxItems: 32768
+          description: The array of column descriptions.
+    CommitInfo:
+      type: object
+      description: Represents a coordinated commit version.
+      properties:
+        version:
+          type: integer
+          format: int64
+          description: The version of this commit.
+        timestamp:
+          type: integer
+          format: int64
+          description: The timestamp for when the commit was made. This is the in-commit timestamp as produced by the Delta client writing to the table.
+        file_name:
+          type: string
+          description: The filename of the UUID-based commit file.
+        file_size:
+          type: integer
+          format: int64
+          description: The size of the commit file in bytes.
+        file_modification_timestamp:
+          type: integer
+          format: int64
+          description: The modification time of the commit file. This is the mod time of the file as written to the file system.
+        is_disown_commit:
+          type: boolean
+          description: Should be set if this commit is a disown commit that removes UC as the commit coordinator from the table.
```

Comment:
What's the meaning of `disown`  ? what's  the behavior if we set this field as true or false ? Could you please explain more ?